homeassistant/components/stream/worker.py

```diff
@@ -452,6 +452,10 @@ def stream_worker(
 ) -> None:
     """Handle consuming streams."""
 
+    if av.library_versions["libavformat"][0] >= 59:
```

Comment:
So the version of PyAV is fixed, but PyAV itself can be built against a few different versions of the ffmpeg libraries. Even if you install the binary wheel, which version you get depends on what platform you're on. The pypi av wheels themselves are built against a fixed ffmpeg version, but the HA alpine wheels get built against the alpine version. Core users are the most troublesome since they can potentially install/build PyAV against any ffmpeg...

Reply:
ok, makes sense.